docs/detections/machine-learning/machine-learning.asciidoc

```diff
@@ -76,6 +76,19 @@ prior to the time they are enabled. After jobs are enabled, they continuously
 analyze incoming data. When jobs are stopped and restarted within the two-week
 time frame, previously analyzed data is not processed again.
 
+[float]
```

Comment:
How about introducing ML integrations as another `Or` after line 67 since these are also considered prebuilt jobs? So saying something like:

```
Or

* You install one or more of the Advanced Analytics integrations.......
```

Reply:
Added a brief line!